pages/battle.php

```diff
@@ -141,11 +141,11 @@ function battle(): bool {
 			}
 			if($player->last_death_ms > System::currentTimeMs() - (60 * 1000)) {
 				throw new Exception("You died within the last minute, please wait " .
-					((($player->last_death_ms + (60 * 1000)) - System::currentTimeMs()) / 1000) . " more seconds.");
+					floor((($player->last_death_ms + (60 * 1000)) - System::currentTimeMs()) / 1000) . " more seconds.");
```

Comment:
```suggestion
					ceil((($player->last_death_ms + (60 * 1000)) - System::currentTimeMs()) / 1000) . " more seconds.");
```
This is probably more intuitive to players, otherwise we could show "please wait 0 seconds"